src/test/java/com/zextras/LdapIT.java

```diff
@@ -0,0 +1,17 @@
+package com.zextras;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.PullPolicy;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+public abstract class LdapIT {
+
+  @Container
+  protected GenericContainer container =
+      new GenericContainer<>("carbonio/ce-ldap-u20:latest")
```

Comment:
We'll have a chat with @M0Rf30 regarding the tagging of Carbonio LDAP containers in order to avoid bad surprises when new commits are released